src/transformers/fiddle-embedder.js

```diff
@@ -53,12 +53,27 @@ async function transformer(tree) {
    */
   function visitor(node, ancestors) {
     const parent = ancestors[0];
-    const folder = node.meta;
+    // Supported formats are fiddle='<folder>|<option>|option'
+    // Options must be of the format key=value with no additional quotes.
+    const [folder, ...others] = node.meta.split("|");
+    const options = {};
+
+    // If there are optional parameters, parse them out to pass to the getFiddleAST method.
+    if (others.length) {
```

Comment:
```suggestion
    if (others.length > 0) {
```
I prefer a more deliberate check here

Reply:
I think Ethan's suggestion makes it more explicit, which is good.